packages/taro-platform-ascf/src/template.ts

```
@@ -0,0 +1,133 @@
+
+  buildXsTemplate (filePath = './utils') {
+    return `<hjs module="xs" src="${filePath}.hjs" />`
+  }
+
+  createMiniComponents (components): any {
+    const result = super.createMiniComponents(components)
+
+    // PageMeta & NavigationBar
+    this.transferComponents['page-meta'] = result['page-meta']
+    this.transferComponents['navigation-bar'] = result['navigation-bar']
+    delete result['page-meta']
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)